mlflow/tracking/fluent.py

```diff
@@ -218,14 +218,33 @@ def start_run(
         import mlflow
 
         # Create nested runs
-        with mlflow.start_run(run_name='PARENT_RUN') as parent_run:
+        experiment_id = mlflow.create_experiment("experiment1")
+        with mlflow.start_run(
+            run_name="PARENT_RUN",
+            experiment_id=experiment_id,
+            tags={"version": "v1", "priority": "P1"},
+            description="starting a parent for experiment {}".format(experiment_id),
```

Comment:
```suggestion
            description="parent",
```

to simplify the example.